doc/custom_rosdistro.rst

```diff
@@ -124,6 +124,17 @@ rosdep database:
 
     rosdep update
 
+To make devel jobs use your custom rosdep database, update the ``targets`` dict
+in the  \<distribution\>/source_build.yaml file in your forked
+ros_buildfarm_config repository with the ``custom_rosdep_urls`` option:
+
+```yaml
+targets:
+  _config:
+    custom_rosdep_urls:
+      - https://raw.githubusercontent.com/YOUR_FORK/rosdistro/master/rosdep/sources.list.d/20-default.list
+```
```

Comment:
Think I have it fixed now. I didn't realize at first that the previewer I was using locally (based on pandoc) doesn't treat reStructuredText in exactly the same way as GitHub does.